lib/beaker-pe/install/pe_utils.rb

```diff
@@ -136,6 +136,14 @@ def remove_client_datadir(host)
           on(host, "rm -rf #{client_datadir}")
         end
 
+        #Return true if tlsv1 rptocol needs to be enforced
+        #param [Host] the host
+        def require_tlsv1?(host)
+          tlsv1_platforms = ["AIX", "el-5", "solaris10"]
+          return true if tlsv1_platforms.any? {|platform| host['platform'] =~ /#{platform}/}
```

Comment:
this is a little bit hard to understand because it will return true if this passes, but rely on the default `nil` return to pass a false result (which will happen) if no match is found.

It would be simpler to say
```ruby
return tlsv1_platforms.any? {|platform| host['platform'] =~ /#{platform}/}
```
or perhaps easier than having regex strings that get interpolated into regex, perhaps you just have a list of regex's & then you don't need interpolation:
```ruby
  tlsv1_platforms = [/AIX/, /el-5/, /solaris10/]
  return tlsv1_platforms.any? { |platform_regex| host['platform'] =~ platform_regex }
```
although the more rubyist way (which I don't agree with) would be to probably not use the `return` keyword & let the method implicitly return the result of the `any?` call.

Reply:
Sounds good, updated